### ISSUE TITLE
Exclude Worker Pools from license limits altogether

### ISSUE THREAD

**Issue by michaelnoonan** [OctopusDeploy/Issues#5046]
Remove Worker Pools from all licensing limits. When we first shipped workers we decided to use [these limitations](https://octopus.com/blog/octopus-release-2018.7#licensing). Since then we've decided to remove Woker Pools from any licensing altogether.

> We don't want to restrict the way people arrange their workers, just the number of workers themselves.

The end result will be similar to Environments which are always unlimited, even if you have a single Deployment Target.

**Comment by octoreleasebot** [comment]
Release Note: Worker pools are no longer limited by licensing

**Comment by ScottGuymer** [comment]
I initially misunderstood this change as removing the restrictions on workers. But after reading carefully I still dont understand why the limit on worker pools has been lifted and what type of license this affects.

We have a grandfathered enterprise license and this change has had no effect for us.

As i understand from the docs

>Customers using a grandfathered Community (free), Professional, Team, or Enterprise license are limited to one worker and worker pool. This enables you to transition from using the built-in worker into a more secure configuration.

Unlimited worker pools here would be useless a you only have one worker..

> Customers using a grandfathered High Availability license get unlimited workers and worker pools.

This change does not apply here

>Customers with one of the current Standard or Data Center licenses also get unlimited workers and worker pools. These licenses were introduced around the Octopus 4.0 timeframe.

This change does not apply here

> Octopus Cloud customers also get unlimited workers and worker pools.

This change does not apply here


Could you shed any light on this?

**Comment by michaelnoonan** [comment]
Hi @ScottGuymer. Thanks for writing. :) The short story is that we originally tied Worker and Worker Pool limits together. We've since realised we don't want to limit Worker Pools in any way, just the Workers themselves. In other words we don't want to restrict anyone from arranging their Workers into whichever groupings they want. This is similar to the way we limit Deployment Targets, but no the way you arrange those into Environments or Roles.

I've updated the [blog post](https://octopus.com/blog/octopus-release-2018.7#licensing) where I think you got those details from originally.

Depending on your scenario, reaching out to sales@octopus.com would be the best way to get things sorted out.

Hope that helps!
Mike
